src/utils/exportServerQueryBuilder.ts

```diff
@@ -46,6 +46,25 @@ export async function probeServer(url: string): Promise<{ output: BulkDataRespon
   } else if (results) {
     throw new Error(`Unexpected response from bulk export server status: ${results.status}`);
   } else {
-    throw new Error('An unknown ocurred while retrieving data from bulk export server');
+    throw new Error('An unknown occurred while retrieving data from bulk export server');
+  }
+}
+
+/**
+ * Handles the GET request when the exportType is static.
```

Comment:
Nitpicking here, but I think this function comment could be a bit more descriptive. Maybe one line to explain what the manifest is and what the function returns